### DISCUSSION TITLE
"eslint kann nicht geladen werden" error

### DISCUSSION THREAD

**Discussion by KlausLex** [VDI-CodING/codeING-main#29] [S1D0] [answered]
Bei mir kommt die Fehlermeldung, dass Skripten aktiviert sein müssen wie hier gezeigt:

https://www.com-magazin.de/tipps-tricks/powershell/windows-10-verweigert-ausfuehrung-powershell-skript-2546684.html

**Comment by stan1025** [comment, marked as answer]
https://www.com-magazin.de/tipps-tricks/powershell/windows-10-verweigert-ausfuehrung-powershell-skript-2546684.html

**Reply by stan1025** [reply]
Danke @KlausLex

**Reply by Philipp1L2P** [reply]
Stark  @KlausLex
